src/registrar/models/domain_information.py

```diff
@@ -212,36 +213,60 @@ def __str__(self):
             return ""
 
     @classmethod
-    def create_from_da(cls, domain_application, domain=None):
-        """Takes in a DomainApplication dict and converts it into DomainInformation"""
-        da_dict = domain_application.to_dict()
-        # remove the id so one can be assinged on creation
-        da_id = da_dict.pop("id", None)
+    def create_from_da(cls, domain_application: DomainApplication, domain=None):
```

Comment:
This refactored function is a little hard to read from the git change editor, I recommend reading this from your code editor